framework/src/org/checkerframework/common/basetype/BaseTypeVisitor.java

```diff
@@ -193,6 +196,9 @@ public BaseTypeVisitor(BaseTypeChecker checker) {
         this.visitorState = atypeFactory.getVisitorState();
         this.typeValidator = createTypeValidator();
         this.vectorType = atypeFactory.fromElement(elements.getTypeElement("java.util.Vector"));
```

Comment:
Can we change these two fields to lazy initialization? Looking up elements and methods is rather costly and this is behavior that gets used very, very seldom.